OpenSim/Sandbox/MatlabScripts/pendulum_simulation.m

```diff
@@ -0,0 +1,86 @@
+%  *                    OpenSim:  testREADME.cpp                                *
+%  * -------------------------------------------------------------------------- *
+%  * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+%  * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+%  * OpenSim is developed at Stanford University and supported by the US        *
+%  * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+%  * through the Warrior Web program.                                           *
+%  *                                                                            *
+%  * Copyright (c) 2005-2016 Stanford University and the Authors                *
+%  * Author(s): James Dunne, Thomas Uchida                                      *
+%  * Contributor(s): Chris Dembia                                               *
+%  *                                                                            *
+%  * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+%  * not use this file except in compliance with the License. You may obtain a  *
+%  * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+%  *                                                                            *
+%  * Unless required by applicable law or agreed to in software                 *
+%  * distributed under the License is distributed on an "AS IS" BASIS,          *
+%  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+%  * See the License for the specific language governing permissions and        *
+%  * limitations under the License.                                             *
+%  * -------------------------------------------------------------------------- */
+
+%% Read a model into memory, run a fwd simulation,
+clear all; close all; clc;
+
+import org.opensim.modeling.*
+
+%   public static void test_TableReporter_2() throws java.io.IOException {
+timeInterval = 0.1;
+
+% add a model
+modelFileName = 'double_pendulum_markers.osim';
+model = Model(modelFileName);
+
+%% add a console reporter
+consoleReporter = ConsoleReporter();
+consoleReporter.set_report_time_interval(timeInterval);
+
+% When connecting the outputs, set the alias names 'pin1_anlge' and 'q2'.
+% These will appear as the colomn labels
+consoleReporter.updInput('inputs').connect(model.getCoordinateSet().get(0).getOutput('value'),'pin1_angle');
+consoleReporter.updInput('inputs').connect(model.getCoordinateSet().get(1).getOutput('value'),'q2');
+model.addComponent(consoleReporter);
+
+%% add a table reporter for the pendulum coordiantes
+coordinateReporter= TableReporter();
```

Comment:
Consider consistent spacing around `=` (earlier, you have a space on both sides).